docs/README.md

```diff
@@ -23,6 +25,46 @@ tbd
 1. Click **Operators** -> **Installed Operators** to verify that the sail-operator 
 is installed. `Succeeded` should appear in the **Status** column.
 
+#### Installing using the CLI
+
+*Prerequisites*
+
+* You must have `admin` privileges.
```

Comment:
What are `admin` privileges? Do you mean the `admin` ClusterRole? Does that suffice or does the user have to have the `cluster-admin` ClusterRole?

Reply:
I guess that because we want to install the operator in the `openshift-operators` namespace, the `cluster-admin` is needed. Going to fix it